docs/calyx-py.md

```diff
@@ -1,17 +1,120 @@
 # Emitting Calyx from Python
 
-Our frontends are written in Python3 and make use of the `calyx` library to
+Our frontends are written in Python3 and make use of the `calyx` builder library to
 generate their code.
 
+## Installation
+
 To install the library, run the following from the repository root (requires
 [flit][] installation):
+
 ```
 cd calyx-py && flit install -s
 ```
 
-The library provides an example:
+## Using the `calyx` builder
+
+The `calyx` library provides a builder to generate Calyx code. Let's walk through the file `calyx-py/test/builder_example.py`. This Calyx program initializes two registers with the numbers 1 and 41, adds them together, and stores the result in a register.
```

Comment:
Link to the corresponding file for `builder_example.py`

Reply:
This will also let you just name the file instead of describing `the/whole/path`